menu.rb

```diff
@@ -0,0 +1,66 @@
+# Breakfast/Bronze program
+
+# # random drink generator created by using a mixer, and juice, and a liquor
+
+# juice = ["cranberry juice","agave nectar", "pineapple juice", "mango nectar", "grapefruit juice"]
+
+# mixer = ["Sprite", "tonic water", "ginger ale", "ginger beer", "club soda"]
+
+# liquor = ["rum", "vodka", "gin", "whiskey","tequila"]
+
+
+# puts "Welcome to the random drink generator. Let's make happy hours at ADA\n more fun
+# here's some drink ideas just to get you started!"
+
+# x = 0
+# 10.times do
+# random_drink = juice[rand(juice.length)] +" with " + mixer[rand(mixer.length) ] +" and " + liquor[rand(liquor.length)]
+# puts "#{x += 1}. How about " + random_drink + "?"
+
+# end
+
+#Random Menu Generator exercise - Silver/Lunch level
+
+# Maybe I have too many bartenders as friends. Don't judge me. But my first thought was to create
+#drink mixer. My initial bronze/breakfast formula as you can see from above was already set to silver/lunch parameters.
+#however, I had to expand my arrays from the general selections I started with because they did not
+#contain enough items.
+
+# random drink generator created by using a mixer, and juice, and a liquor
+
+# juice, mixer, and liquor are all arrays, which contain a liquid assigned to a certain place value
+# within each array. e.g Cranberry juice holds the [0] place in the juice array.
+
+juice = ["cranberry juice","agave nectar", "pineapple juice", "mango nectar", "grapefruit juice",
+ "orange juice","pomegranate juice", "pear nectar", "peach nectar", "coconut water" ]
```

Comment:
Try to be consistent with your array spacing and indenting. It might seem unimportant, but it can have a pretty big impact on the readability of your code